### ISSUE TITLE
intergrate `baidu` web search for Chinese users

### ISSUE THREAD

**Issue by u-Sir** [adirh3/Fluent-Search#758]
Google is blocked in China. And `baidu` is a common search in China

baidu search URL: https://www.baidu.com/s?wd=
baidu suggestion URL: https://suggestion.baidu.com/su?wd=%s

**Comment by Jefferycheng** [comment]
You could customize your web search in settings.

Path : settings -> Web -> Web search tags

Click the '+' on the right side, and fill in the following fields Name, Tag, and URL.

And then, you can get the search result in your search.

**Comment by u-Sir** [comment]
> You could customize your web search in settings.
> 
> Path : settings -> Web -> Web search tags
> 
> Click the '+' on the right side, and fill in the following fields Name, Tag, and URL.
> 
> And then, you can get the search result in your search.

yeah, I know this. And I hope integrate it for normal users in Chinese.
